plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -50,7 +50,7 @@ function defaultShouldPreventSpanCreation() {
  * addEventListener of HTMLElement
  */
 export class UserInteractionInstrumentation extends InstrumentationBase {
-  readonly version = VERSION;
+  readonly version = PACKAGE_VERSION;
```

Comment:
I think we should also update the `moduleName` here on the next line to be `PACKAGE_NAME`. It gets used in the `enable()` and `disable()` methods.

Reply:
This suggested change looks right to me, but it will introduce a change to the instrumentation (it's `name`) which is probably ok but I don't want to bundle it into this PR, as it will change the scope from `refactor` to `fix`.